src/condor_credd/condor_credmon_oauth/condor_credmon_oauth

```diff
@@ -16,8 +17,13 @@ import htcondor2 as htcondor
 import sys
 sys.path.append(htcondor.param.get('libexec', '/usr/libexec/condor'))
 
-from credmon.CredentialMonitors.OAuthCredmon import OAuthCredmon
+from credmon.CredentialMonitors.ClientCredmon import ClientCredmon
 from credmon.CredentialMonitors.LocalCredmon import LocalCredmon
+from credmon.CredentialMonitors.OAuthCredmon import OAuthCredmon
+try:
+    from credmon.CredentialMonitors.VaultCredmon import VaultCredmon
+except ImportError:
+    VaultCredmon = None
```

Comment:
All four of these are optional in `__init__.py`; why are three of them required here?  Especially consider making `ClientCredmon` optional, since it's experimental code.

Reply:
Cleaned up in 866d438789662139287bdceda6d41ae81414f8f4